espflash/src/cli/mod.rs

```diff
@@ -133,7 +133,7 @@ pub struct FlashArgs {
     /// Logging format.
     #[arg(long, short = 'L', default_value = "serial", requires = "monitor")]
     pub log_format: LogFormat,
-    /// Minimum chip revision supported by image, in format: major * 100 + minor
+    /// Minimum chip revision supported by image, in format: major.minor
     #[arg(long, default_value = "0", value_parser = parse_chip_rev)]
```

Comment:
ah sorry - what I meant was `default_value = "0.0"` since otherwise when not providing the argument we get

```
error: invalid value '0' for '--min-chip-rev <MIN_CHIP_REV>': Failed to parse chip revision: 0. Chip revision must be in the format `major.minor`
```

Reply:
Oh! Didn't notice it, updated.